src/private_cuda.h

```diff
@@ -46,6 +46,24 @@
 /* Keep in sync with the copy in gpuarray/extension.h */
 #define DONTFREE 0x10000000
 
+#define GA_CUDA_EXIT_ON_ERROR(ctx, cmd) \
+  do {                                  \
+    int err = (cmd);                    \
```

Comment:
Ok I am fixing this. Compiling with "-Wdeclaration-after-statement" gives also:

```
/home/tsirif/Development/libgpuarray/src/gpuarray_blas_cuda_cublas.c: In function ‘dgemmBatch’:
/home/tsirif/Development/libgpuarray/src/gpuarray_blas_cuda_cublas.c:686:3: warning: ISO C90 forbids mixed declarations and code [-Wdeclaration-after-statement]
   const size_t threshold = 650;
```

What should be done with const declarations?